Documentation/op-guide/_index.md

```diff
@@ -1,3 +1,4 @@
 ---
 title: Operations guide
+weight: 4000
```

Comment:
@nate-double-u - missing description here?  looks great otherwise. Thanks!

Reply:
Oops, thanks! I'll write something up for that 😅️

Reply:
Updated